README.md

```diff
@@ -1,71 +1,91 @@
 Accoutrement-Type
 =================
 
-Sass typography [Accoutrement][accoutrement]
+Sass typography and webfont helpers
 by [OddBird][oddbird].
 Gather all your fonts into a single map,
 access them by name,
 and easily import the webfonts.
 Also includes helpers for accessibility
 and pseudo-elements.
 
-[accoutrement]: http://oddbird.net/accoutrement/
 [oddbird]: http://oddbird.net/
 
 
+More Accoutrement Tools
+-----------------------
+
+- [Init](http://oddbird.net/accoutrement-init/)
+  provides light-weight browser-normalization.
+- [Color](http://oddbird.net/accoutrement-color/)
+  provides color-palette management and contrast-ratio utilities.
+- [Scale](http://oddbird.net/accoutrement-scale/)
+  helps manage scale patterns like font-sizes, margins, and gutters.
+- [Layout](http://oddbird.net/accoutrement-layout/)
+  provides layout utilities such as
+  box-sizing, intrinsic ratios, z-index management,
+  named media-queries, and a clearfix.
+
+
 Quick Start
 -----------
 
+Install the package with npm or yarn:
+
 ```bash
 npm install accoutrement-type
+yarn add accoutrement-type
 ```
 
 Import the library:
 
 ```scss
-@import 'path/to/accoutrement-type/type';
+@import '<path-to>/accoutrement-type/sass/type';
 ```
 
 Configure your font settings:
 
 ```scss
-$font-formats: 'woff' 'ttf'; // Define what webfont formats need importing
-$font-path: '../fonts/'; // Set the a path to your fonts directory
+// Define what webfont formats need importing
+$font-formats: 'woff' 'ttf';
+
+// Set the a path to your fonts directory
+$font-path: '../fonts/';
 
 $fonts: (
-  'heading': ( // give your font a semantic name for reference
-    'name': 'maven', // optionally set a different font name
-    'stack': ('helvetica', 'arial', sans-serif), // define the stack
-    'normal': 'maven/maven_pro_regular-webfont', // point to any webfont files
+  // describe locally-hosted font-files for import and access
+  'heading': (
+    'name': 'maven',
+    'stack': ('helvetica', 'arial', sans-serif),
+    'normal': 'maven/maven_pro_regular-webfont',
     'bold': 'maven/maven_pro_bold-webfont',
   ),
 
+  // describe CDN fonts for access
   'body': (
-    'name': 'exo',
-    'stack': ('helvetica', 'arial', sans-serif),
-    'normal': false, // set variant paths to false when using font CDNs...
-    'italic': false, // ...if you still want to document available styles
-    'bold': false,
-    'bold' 'italic': false,
+    'name': 'Source Sans Pro',
+    'source': 'https://fonts.google.com/specimen/Source+Sans+Pro',
```

Comment:
I wonder if this will be confusing since Google will give you a URL with the font file weight in it. 
`'https://fonts.googleapis.com/css?family=Open+Sans:300,400,700'` is the url it tells you to use. 
![screen shot 2017-09-12 at 1 31 36 pm](https://user-images.githubusercontent.com/1581694/30342234-c2daae88-97be-11e7-8d00-f3cb4d28ab8f.jpg)

Reply:
Since `source` is actually a Herman feature ignored by accoutrement-type, I'll remove it here. We can document it in Herman.

Reply:
Adding to the `$fonts` documentation…

```
/// Safely provide any extra data you need access to,
/// since accoutrement-type will ignore unknown font properties.
/// for example,
/// we've added a `source` property to our fonts,
/// providing a link to the font foundry, license, or CDN.
/// If you are using [Herman][http://oddbird.net/herman/]
/// to generate style-guides and font previews,
/// it will provide any listed `source` links in the output.
```

Reply:
👍